### PR TITLE
small fixes for tutorials

### PR DESCRIPTION
**Pull request by skoudoro**
A quick PR that fixes some bad rendering on different tutorials

### CODE REVIEW

File: tools/ex2rst [hunk 1]
```diff
@@ -96,7 +96,8 @@ def exfile2rst(filename):
 
         # if we have something that should go into the text
         if indocs \
-           or (cleanline.startswith('"""') or cleanline.startswith("'''")):
+           or (cleanline.startswith('"""') or cleanline.startswith("'''") or
+               cleanline.startswith('r"""') or cleanline.startswith("r'''")):
```
Comment (arokem) [inline tools/ex2rst:100]: Good one!

**Comment by arokem** [comment]
This looks good! Looks like the new version of nibabel is causing some issues.

**Comment by skoudoro** [comment]
All good! Can you merge this PR @arokem? @Garyfallidis?

**Comment by skoudoro** [comment]
that was fast! Thanks!

**Pull request merged**